en/docs/administer/logging-and-monitoring/monitoring/working-with-observability.md

```diff
@@ -774,4 +774,4 @@ In order to enable blacklisting of threads, add the following configuration as a
 
 If the above configuration is not added, by default the `MessageDeliveryTaskThreadPool` thread will be blacklisted as it is found to print a considerable amount for messages for API-M instances. However, if the above configuration is added the default value will be overridden. 
 
-Blacklisting of threads is not needed by default, as all unnecessary threads are already blacklisted.
\ No newline at end of file
+Blacklisting of threads is not needed by default, as all unnecessary threads are already blacklisted.
```

Comment:
@Arshardh & @bhathiya , isn't it incorrect to use the term blacklisting in 3.2.0 docs?

Reply:
@Arshardh is this a config?